rebar.lock

```diff
@@ -1,7 +1,7 @@
 {"1.1.0",
 [{<<"aebytecode">>,
   {git,"https://github.com/aeternity/aebytecode.git",
-      {ref,"56cf62b48753a61e30b81d805eee5ec36959b688"}},
+      {ref,"1526ad3bf057e72a1714aea0430b001bd1d576c9"}},
```

Comment:
I see https://github.com/aeternity/aebytecode/compare/56cf62b48753a61e30b81d805eee5ec36959b688...1526ad3bf057e72a1714aea0430b001bd1d576c9

Reply:
It only adds in `include/aeb_opcodes.hrl`:
```
-define(PRIM_CALL_CRYPTO_ECVERIFY_SECP256K1, 410).
```

(There was already defined:
```
-define(PRIM_CALL_IN_CRYPTO_RANGE(__TTYPE__), (((__TTYPE__) > 399) andalso ((__TTYPE__) < 500))).
```
.)